api/database/migrations/2025_09_30_133328_create_personal_access_tokens_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
```

Comment:
@majamitic774 This file is duplicated. So when we run docker up laravel will run both mirations and try to create same table twice. This will cause an error.